rbac/management/role/view.py

```diff
@@ -349,3 +363,17 @@ def validate_and_get_access_list(self, data):
         for access in access_list:
             AccessSerializer(data=access).is_valid(raise_exception=True)
         return access_list
+
+    def validate_and_get_additional_field_key(self, params):
+        """Validate the add field key."""
+        field = params.get(ADDITIONAL_FIELD_KEY)
```

Comment:
Fixed, now it supports add_fields=groups_in_count+groups_in

Reply:
Hey @astrozzc. It doesn't appear there's an actual web spec standard, but the consensus seems to be to user a comma delimiter to represent an OR operator, and & for an AND operator. So in this case it would probably make more sense to use a comma, to be able to have: `?add_fields=groups_in_count,groups_in`?

https://stackoverflow.com/a/24728298/1331784

Reply:
Right, that's what I tried first using comma, but I keep getting error of invalid input. Let me try to find the issue why comma is not working

Reply:
I found the reason, I put an extra space in between` ?add_fields=groups_in_count, groups_in?`, no wonder it is failing. Updated. Now it looks much better 👍